env0/data_environment.go

```diff
@@ -150,6 +150,24 @@ func dataEnvironmentRead(ctx context.Context, d *schema.ResourceData, meta inter
 		return diag.FromErr(err)
 	}
 
+	// Set this explicitly because these are not set in setEnvironmentSchema due to possible project inheritance.
```

Comment:
Can we have a test validating the issue was fixed? basically using the same code that is described in the issue, and validating 2nd apply doesn't have a drift

Reply:
Added acceptance test:
91687ecb46de61b82995c763267d20be3d5ce588